firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -3399,7 +3399,7 @@ extension BrowserViewController: UIAdaptivePresentationControllerDelegate {
 
 extension BrowserViewController {
     /// Used to get the context menu save image in the context menu, shown from long press on webview links
-    fileprivate func getImageData(_ url: URL, success: @escaping (Data) -> Void) {
+    func getImageData(_ url: URL, success: @escaping (Data) -> Void) {
```

Comment:
I removed the `fileprivate` access modifier to use this function in the builder. While I'm concerned about the potential misuse of this function in unintended places, the implementations remain the same. What do you think?